game/script-topics.rpy

```diff
@@ -294,22 +312,283 @@ label talk_get_picture_permission:
+                n "I wanna hear all about it!"
+
+            "I used to.":
+                n "Oh...{w=0.3} oh gosh."
+                n "I'm really sorry to hear that,{w=0.1} [player]."
+                n "I hope you're managing okay now."
+                n "..."
+                n "I...{w=0.3} think we should talk about something else, alright?"
+
+    else:
+        n "What about you,{w=0.1} [player]?"
+        n "Did you get another one?"
+        menu:
+            "Yes, I did.":
+                n "Ooh...{w=0.3} you gotta tell me!{w=0.2} What did you get?"
+                call pet_options_a
+
+            "No, I didn't.":
+                n "Oh.{w=0.2} Well,{w=0.1} that's fair."
+                n "You're already giving a home to something,{w=0.1} so I won't complain!"
+
+            "I lost one.":
+                n "Oh...{w=0.3} oh jeez..."
+                n "I'm so sorry,{w=0.1} [player].{w=0.2} Are you okay?"
+                n "Maybe we should talk about something else to keep your mind off things..."
+                n "I'm here for you,{w=0.1} [player]."
+
+    return
+
+label pet_options_a:
+    menu:
+        "Birds":
+            n "Oh!{w=0.2} Neat!"
+            n "I don't think I'd keep birds myself,{w=0.1} but they brighten up rooms for sure!"
+            n "It doesn't get too noisy for you,{w=0.1} I hope?"
+            n "I'm sure yours appreciate your company though."
+            $ persistent.jn_player_pet = "birds"
+
+        "Cats":
+            n "Yay!{w=0.2} Cats!"
+            n "I really wish I had one,{w=0.1} I love seeing all the dumb situations they get into!"
+            n "I hope you didn't just say that because I like them,{w=0.1} though.{w=0.1} Ehehe."
+            n "Just don't pamper it too much,{w=0.1} [player]!"
+            $ persistent.jn_player_pet = "cats"
+
+        "Dogs":
+            n "Oh!{w=0.2} A dog?{w=0.2} Awesome!"
+            n "I don't think a dog would be my first choice,{w=0.1} what with all the walks and all that."
+            n "But I can't think of a more loving pet!"
+            n "I hope yours looks after you as much as you look after it!"
+            $ persistent.jn_player_pet = "dogs"
+
+        "Fish":
+            n "Ooh!{w=0.2} Fish are interesting!"
+            n "I don't think I'd call them super affectionate personally..."
+            n "But I think they're a neat way to relieve stress!{w=0.2} They must be calming to watch in their own little world."
+            n "I bet you feel like you could lose yourself in that tank!{w=0.2} Ehehe."
+            $ persistent.jn_player_pet = "fish"
+
+        "Gerbils":
+            n "Awww!{w=0.2} I like gerbils!"
+            n "It's so cute how they live in little groups to keep each other company."
+            n "They're good at digging,{w=0.1} too -{w=0.2} like seriously good!"
+            n "Take good care of yours for me,{w=0.1} okay?"
+            $ persistent.jn_player_pet = "gerbils"
+
+        "More...":
```

Comment:
(just for future reference - this should be removed once new menus are in place)

Reply:
Will do!